keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlResourceRepository.kt

```diff
@@ -114,6 +117,7 @@ class SqlResourceRepository(
         field("kind") to resource.kind,
         field("name") to resource.metadata.name.value,
         field("resource_version") to resource.metadata.resourceVersion,
+        field("metadata") to objectMapper.writeValueAsString(resource.metadata.data),
```

Comment:
this is just the extra data bits of the metadata, currently we would drop those on the floor